frontend/src/scenes/insights/InsightsTable/InsightsTable.tsx

```diff
@@ -159,10 +161,10 @@ export function InsightsTable({
                         onLabelClick={canEditSeriesNameInline ? () => handleEditClick(item) : undefined}
                     />
                     {canEditSeriesNameInline && (
-                        <EditOutlined
-                            title="Rename graph series"
-                            className="edit-icon"
+                        <LemonButton
                             onClick={() => handleEditClick(item)}
+                            title="Rename graph series"
+                            icon={<IconEdit className="edit-icon" />}
```

Comment:
The hover area of this one is a bit odd because of `.series-name-wrapper-col .edit-icon` styling:
<img width="131" alt="Screen Shot 2022-03-14 at 13 18 46" src="https://user-images.githubusercontent.com/4550621/158170703-26b1a2c1-cc22-44a1-8174-3ab8de56a324.png">